tests/fuzz/app.js

```diff
@@ -0,0 +1,72 @@
+var express = require('express');
+
+var redis = require('redis'),
+    client = redis.createClient();
+var async = require('async');
+
+var xssFilters = require('../../dist/xss-filters');
+
+var app = express();
+
+function buildResponse(id, fuzz) {
+  return "<html> <body> <h1> Fuzz Test! </h1>" +
+
+  "<script>window.alert = function(msg) { console.log(\"" + id + ":\"+ msg) } </script>" +
+  "<script>window.prompt = function(msg) { console.log(\"" + id + ":\"+ msg) } </script>" +
+
+  // "<script>alert('xssssss')</script>" +
+  // "<SCRIPT SRC='http://yahoo.com'></SCRIPT>" +
+
+  "<div style='border: 1px solid'> " +
+    "<h2> xssFilters.inHTMLData(fuzz) </h1>" +
+    "<div> " + xssFilters.inHTMLData(fuzz) + " </div> " +
+  "</div>" +
+
+
+  "<div style='border: 1px solid'> " +
+  "<h2> xssFilters.inHTMLComment(fuzz) </h2>" +
+  "<!-- " + xssFilters.inHTMLComment(fuzz) + " --> " +
+  "</div>" +
+
+
+  "<div style='border: 1px solid'> " +
+  "<h2> xssFilters.inSingleQuotedAttr(fuzz) </h2> " +
+  "<input value='" + xssFilters.inSingleQuotedAttr(fuzz) + "'/>" +
+  "</div>" +
+
+
+  "<div style='border: 1px solid'> " +
+  "<h2> xssFilters.inDoubleQuotedAttr(fuzz) </h2> "+
+  "<input value=\"" + xssFilters.inDoubleQuotedAttr(fuzz) + "\"/>" +
+  "</div>" +
+
+
+  "<div style='border: 1px solid'> " +
+  "<h2> xssFilters.inUnQuotedAttr(fuzz) </h2> " +
+  "<input value=" + xssFilters.inUnQuotedAttr(fuzz) + "'/>" +
+  "</body> </html>";
+}
+
```

Comment:
Yeah, at first I figured I'd test the ones that are simplest to trigger with an XSS with. I'll look into adding the others.